src/templates/app/user/metatrader.jsx

```diff
@@ -156,7 +156,7 @@ const Metatrader = () => (
                         <div id='account_desc' className='mt-container border-bottom' />
                     </div>
                     <div className='mt-panel'>
-                        <div className='mt-sidebar-button border-bottom'>
+                        <div className='mt-sidebar-button border-bottom' id='mt-ea'>
```

Comment:
we have a property data-hide, you can use it to hide this section for malta and maltainvest, it doesn't work in this case?